### PR TITLE
Merge from internal

### PR DESCRIPTION
**Pull request by cynthiajoan**
[replace this line]: # (Describe your changes in detail.)
***

[replace this line]: # (Describe your testing in detail.)
***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [ ] Other, such as a build process or documentation change.
***

### CODE REVIEW

File: app/src/cpp_instance_manager.h [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include <unordered_map>
 
 #include "app/src/assert.h"
+#include "app/src/include/firebase/internal/mutex.h"
```
Comment (a-maurice) [inline app/src/cpp_instance_manager.h:26]: This is adding a header that is already there.

File: crashlytics/src/cpp/common/crashlytics.cc [hunk 2]
```diff
@@ -21,6 +21,7 @@
 #include "app/src/assert.h"
 #include "app/src/cleanup_notifier.h"
 #include "app/src/include/firebase/app.h"
+#include "app/src/include/firebase/internal/mutex.h"
```
Comment (a-maurice) [inline crashlytics/src/cpp/common/crashlytics.cc:24]: Same here

Comment (cynthiajoan) [reply crashlytics/src/cpp/common/crashlytics.cc:24]: remove the original one to keep the alphabetical sequence

File: app/src/cpp_instance_manager.h (hunk 1 above)
Comment (cynthiajoan) [reply app/src/cpp_instance_manager.h:26]: remove the original one to keep the alphabetical sequence

**Review by a-maurice — APPROVED** [review]

**Pull request merged**